search/search_service/proxy/es_mappings.py

```diff
@@ -0,0 +1,75 @@
+# Copyright Contributors to the Amundsen project.
+# SPDX-License-Identifier: Apache-2.0
+
+from elasticsearch_dsl import (
+    Date, Document, Keyword, Long, Text, normalizer,
+)
+
+column_names_normalizer = normalizer(name_or_instance='column_names_normalizer',
+                                     type='custom',
+                                     filter=['lowercase'])
+lowercase_normalizer = normalizer(name_or_instance='lowercase_normalizer',
+                                  type='custom',
+                                  char_filter=[],
+                                  filter=['lowercase', 'asciifolding'])
+feature_lowercase_normalizer = normalizer(name_or_instance='lowercase_normalizer',
+                                          type='custom',
+                                          char_filter=[],
+                                          filter=['lowercase', 'asciifolding'])
+
+
+class Table(Document):
+    name = Text(fields={'raw': Keyword()}, analyzer='simple')
+    schema = Text(fields={'raw': Keyword()}, analyzer='simple')
+    display_name = Keyword()
+    last_updated_timestamp = Date(format='epoch_second')
+    description = Text(analyzer='simple')
+    column_names = Text(fields={'raw': Keyword(normalizer=column_names_normalizer)},
+                        analyzer='simple')
+    column_descriptions = Text(analyzer='simple')
+    tags = Keyword(multi=True)
+    badges = Keyword(multi=True)
+    cluster = Text(fields={'raw': Keyword()}, analyzer='simple')
+    database = Text(fields={'raw': Keyword()}, analyzer='simple')
+    key = Keyword()
+    total_usage = Long()
+    unique_usage = Long()
+    programmatic_descriptions = Text(analyzer='simple', multi=True)
+
+
+class Dashboard(Document):
```

Comment:
we need to add key to dashboard mapping, currently cant update dashboard because of this https://github.com/amundsen-io/amundsen/blob/main/frontend/amundsen_application/api/metadata/v0.py#L447 @dkunitsk